pkg/lightning/config/config.go

```diff
@@ -1166,6 +1178,31 @@ func (t *TikvImporter) adjust() error {
 		storageSizeDir := filepath.Clean(t.SortedKVDir)
 		sortedKVDirInfo, err := os.Stat(storageSizeDir)
 
+		switch {
+		case os.IsNotExist(err):
+		case err == nil:
+			if !sortedKVDirInfo.IsDir() {
+				return common.ErrInvalidConfig.
+					GenWithStack("tikv-importer.sorted-kv-dir ('%s') is not a directory", storageSizeDir)
+			}
+		default:
+			return common.ErrInvalidConfig.Wrap(err).GenWithStack("invalid tikv-importer.sorted-kv-dir")
+		}
+	case BackendRemote:
+		if t.ParallelImport && t.AddIndexBySQL {
+			return common.ErrInvalidConfig.
+				GenWithStack("tikv-importer.add-index-using-ddl cannot be used with tikv-importer.parallel-import")
+		}
+		if len(t.RemoteWorkerAddr) == 0 {
+			return common.ErrInvalidConfig.GenWithStack("tikv-importer.remote-worker-addr must not be empty!")
+		}
+
+		if len(t.SortedKVDir) == 0 {
```

Comment:
remote backend will not require this folder? It's used for local backend.

Or if it really has an usage, please add comment to explain